modules/admin_manual/pages/installation/manual_installation/server_prep_ubuntu_18.04.adoc

```diff
@@ -33,6 +35,113 @@ The following command installs the Apache web server and PHP.
 sudo apt install php libapache2-mod-php apache2
 ----
 
+If you want to install in two steps, first PHP and then Apache, run the following commands:
+
+[source,console]
+----
+sudo apt install php-cgi
+sudo apt install php
+sudo apt install apache2 libapache2-mod-php
+----
+
+Use `php-fpm` when opting for the FastCGI Process Manager. This requires a non standard setup.
+
+== Multiple Concurrend PHP Versions
+
+If you have multiple concurrent PHP versions installed, which will happen when using `ondrej/php`,
+you need to tell your Web Server and your CLI environment which one to use. 
+To list all available versions and choose one from them, use the following command:
+
+[source,console]
+----
+sudo update-alternatives --config php
+----
+Here is an example output:
+
+[source,console]
+----
+There are 2 choices for the alternative php (providing /usr/bin/php).
+
+  Selection    Path             Priority   Status
+------------------------------------------------------------
+  0            /usr/bin/php7.4   74        auto mode
+* 1            /usr/bin/php7.3   73        manual mode
+  2            /usr/bin/php7.4   74        manual mode
```

Comment:
This example does not have PHP 7.2 in the list at all. I have a feeling that Ubuntu 18.04.5 might nowadays already provide something later than 7.2 - but I would need to install a VM to really find out.

Reply:
You are totally right regaring php. But the thing is a bit more complicated.
I am planning a 20.04 document which will contain an upgrade and new install section.
With 7.4+ there is a special case that needs coverage. I have prepared some stuff already but the basis is this document when merged. There will be a reference for php7.4 to 20.04 to avoid double content and 18.04 is already replaced by 20.4. Means, that 18.04 will get a small addon change but it is easier for me to do this in two steps because I can set the cross references properly.

Therefore, if you are fine with the changes made here, please approve and merge. I will do the backports to 10.6 and 10.5 and then I can start working on 20.04 😄

Reply:
Note: I tried a fresh Ubuntu 18.04.5 in a VM. `sudo apt install php` installs PHP 7.4.3 built May 26 2020. For PHP, Ubuntu has a bit of a moving target. At the time of the original Ubuntu 18.04 release it installed PHP 7.2. So some people with an "old existing" Ubuntu 18.04 might have PHP 7.2 or 7.3, depending when they first installed Ubuntu 18.04. The "long term support" (LTS) for Ubuntu is about the kernel and tightly-linked OS stuff. Versions of various programming languages like PHP still move upwards... (I guess that is because in the modern world there is no way any install should stick with the current version of PHP for 5 years)

So it makes writing these sort of docs more difficult.

But actually, for a fresh Ubuntu 18.04, the above is what happens - "auto mode" shows 7.4 and the instructions here end up retro-fitting 7.3